src/model/bundle.js

```diff
@@ -24,22 +26,57 @@ function create(data) {
         });
 }
 
+function update(id, data) {
+    if(!id || !data || _.isEmpty(data)) {
```

Comment:
`_.isEmpty(data)` can handle case falsy value of data, so it could be 

`if(!id || _.isEmpty(data))`